.travis.yml

```diff
@@ -22,6 +22,8 @@ script:
 
 deploy:
   provider: script
-  script: ./scripts/push.sh
+  script: ./scripts/push.sh production $TRAVIS_TAG
+  skip_cleanup: true
   on:
-    tags: false
+    branch: master
+    tags: true
```

Comment:
I tried manually to release by github to my example code . 
It
Pushed in two case
#1 Once At release time .

 #2 Once when we make changes in master